__tests__/app.test.js

```diff
@@ -242,3 +254,89 @@ describe("POST /api/articles/article_id:/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/article_id:", () => {
+  test("200: Serves the article with the incremented votes", () => {
+    return db
+      .query(`SELECT votes FROM articles WHERE article_id = 1`)
+      .then(({ rows }) => {
+        return request(app)
+          .patch("/api/articles/1")
+          .send({
+            inc_votes: 27,
+          })
+          .expect(200)
+          .then(({ body: { article } }) => {
+            expect(article).toMatchObject({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: rows[0].votes + 27,
+              article_img_url: expect.any(String),
+            });
+          });
+      });
+  });
+  test("200: Appends the new vote count to the database", () => {
+    return db
+      .query(`SELECT votes FROM articles WHERE article_id = 3`)
+      .then(({ rows }) => {
+        return request(app)
+          .patch("/api/articles/3")
+          .send({
+            inc_votes: 48,
+          })
+          .expect(200)
+          .then(() => {
+            return db
+              .query(`SELECT * FROM articles WHERE article_id = 3`)
+              .then(({ rows: [article] }) => {
+                expect(article).toMatchObject({
+                  article_id: expect.any(Number),
+                  title: expect.any(String),
+                  topic: expect.any(String),
+                  author: expect.any(String),
+                  body: expect.any(String),
+                  created_at: expect.any(Date),
+                  votes: rows[0].votes + 48,
+                  article_img_url: expect.any(String),
+                });
+              });
+          });
+      });
+  });
```

Comment:
Nice test, but this isnt really needed, as we should assume if the value has been updated in the first test, it is as a result of a db query and the db has been updated